README.md

```diff
@@ -125,3 +125,19 @@ You can test only specific files by:
 $ npx check-dts **/*.tsx?
 $ npx check-dts **/*.ts !**/index.ts
 ```
+
+## FAQ
+
+### It seems that check-dts ignores all d.ts-files. How can I fix this?
+
+Please make sure that your tsconfig.json doesn't include `skipLibCheck: true`.
+Becuase with this [option](https://www.typescriptlang.org/tsconfig#skipLibCheck) all d.ts-files will be ignored.
```

Comment:
Keep 80 columns line. For instance, by moving link’s URL under the text (check Autoprefixer docs).

Reply:
done! :ok_hand: